### DISCUSSION TITLE
PhotoDemon 9.0

### DISCUSSION THREAD

**Discussion by tannerhelland** [tannerhelland/PhotoDemon#430] [News]
After two years of work, PhotoDemon 9.0 is now available.

Everything new and improved is described in (excruciating) detail in the [release announcement at photodemon.org](https://photodemon.org/2022/09/08/photodemon-9-0.html).

Zip file checksums:

CRC-32: 9a02f3dd
MD5: a9300579dce957a1728231af09da6a90
SHA-1: 27f0be2d419cf87e0c7bcb5bd85062062b73713d
SHA-256: 1917e0594b2544c696abfaace3400370472add5646a5851a5816b2d83e4787d4

<hr /><em>This discussion was created from the release <a href='https://github.com/tannerhelland/PhotoDemon/releases/tag/v9.0'>PhotoDemon 9.0</a>.</em>

**Comment by jpbro** [comment]
Congratulations @tannerhelland, an impressive milestone! I haven't had a chance to play around with the official 9.0 release yet, but I've just installed it and I am looking forward to giving it a spin. Thank you for all of your work on this fantastic program!

**Reply by tannerhelland** [reply]
Thank you for the kind words, @jpbro .  I am very grateful for all your help and support with this release ❤️
